docker/README.md

```diff
@@ -0,0 +1,33 @@
+# Benchmarks in Docker
```

Comment:
For perf testing, each container needs to be run on a separate VM.  We talked about using Docker Swarm to start the whole cluster, but it didn't seem necessary yet.  It's easy enough to build and run the images on each host VM as needed.